### ISSUE TITLE
Accessibility - Table headers sort and row icons

### ISSUE THREAD

**Issue by tappyy** [matfish2/vue-tables-2#708]
Hi there,

We are using VueTables2 in an application where accessibility is vital.

We are using sort-able columns with icons/buttons in each row (stars to add to favourite etc). 

It does not look possible to access the table headers using tab to be able to trigger the sort functionality using a keyboard, or access content of the rows to trigger our favourite buttons.

Is this a known issue? Can a solution be recommended to us?

- Vue.js version: `vue@2.5.16`

- steps for reproducing the issue: Create a `v-client-table` with sort-able columns and use tab to navigate the page. Table headings (and the sort-able functionality) cannot be accessed via tab.

**Comment by jeffcj** [comment]
@tappyy There doesn't appear to be an easy solution for this from a vue-client-table perspective, but there is a workaround to at least make the heading a "link" which prompts a tab stop. You can simply create a custom header for the column that calls a sort function (which is then delegated to the vue-client-table's orderByColumn method. This will allow for tabbing to the column header and pressing Enter to sort.

[codesandbox.io/s/vuetables2-demo-ei6bs](https://codesandbox.io/s/vuetables2-demo-ei6bs)

**Comment by joelmandell** [comment]
Alternative way is to:

```
  options: {
      headings: {
        name: 'Country Name',
        code: function(h) {
        	return h("a",{domProps:{href:"#",innerHTML:"Country code"}})
        },
   },
```

**Comment by joelmandell** [comment]
Sandbox demo with render function defined on the method property.
https://codesandbox.io/s/92bl1

**Comment by matfish2** [comment]
partly solved by #736. Still need to trigger sorting on enter when on sortable column header
